kitsune/products/jinja2/products/includes/topic_macros.html

```diff
@@ -34,15 +34,15 @@ <h3 class="support-callout-title"><span class="highlight">{{ _('Ask the Communit
       </div>
     {% endif %}
 
-    {% if show_contact_form %}
+    {% if show_contact_form and settings.OIDC_ENABLE %}
```

Comment:
Why do we need `OIDC_ENABLE` here? Isn't it the default behaviour?

Reply:
Because it doesn't make sense to have this link enabled in case of a failover to Frankfurt because users will not be able to login (read-only mode).

Reply:
:+1: